admin/views/class-yoast-feature-toggles.php

```diff
@@ -86,6 +86,17 @@ protected function load_toggles() {
 				'read_more_url'   => 'https://yoa.st/2ao',
 				'order'           => 20,
 			],
+			(object) [
+				'name'                => __( 'Inclusive language analysis', 'wordpress-seo' ),
+				'premium'             => true,
+				'in_beta'             => true,
+				'supported_languages' => [ 'en' ],
```

Comment:
We define this as well in `src/helpers/language-helper.php` below. Is there a way to have a single point of definition?